grol_tests/constants.gr

```diff
@@ -1,31 +1,4 @@
+// - Test constants
 
-/*
-NoErr = macro(expr,a,b) {
-    quote(
-		r = catch(unquote(expr))
-	    if r.err {
-			error("unexpected error:", r.value)
-			unquote(a)
-		} else {
-			unquote(b)
-		}
-	)
-}
-*/
-
-if (r=catch(PI)).err {
-	error("unexpected error:", r.value)
-} else {
-	println("PI is:", r.value)
-}
-
-
-if !(r=catch(PI++)).err {
-	error("expected error, got ok", r)
-} else {
-	if regexp("attempt to change constant PI", r.value) {
-		println("Got expected error:", r.value)
-	} else {
-		error("unexpected error:", r.value)
-	}
-}
+NoErr("PI should exist", PI, "3.141592")
+IsErr("Can't change a constant", PI++, "attempt to change constant PI")
```

Comment:
I think that's pretty nice way of making a bunch of tests, thoughts @ccoVeille ?

It's msg, expression, regexp